pkg/controller/config/config_controller_test.go

```diff
@@ -124,8 +124,10 @@ func TestReconcile(t *testing.T) {
 	}
 
 	cs := watch.NewSwitch()
-	tracker, err := readiness.SetupTracker(mgr)
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	tracker := &readiness.Tracker{
```

Comment:
What are we fixing by removing the call to SetupTracker?

Reply:
In this case, the test failed because when the readiness tracker tried to run it tried to pull in `ConstraintTemplate` and would fail because it wasn't a registered schema in this test (or at least that's how I interpreted it). Either way I didn't think it was pertinent to the test, so I just initialized the struct manually.

Reply:
It's likely best for code isolation if we can use constructors. What was the error that you were seeing? Since the existing code works, I'm guessing something got removed?